.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/ChineseDatePeriodParserConfiguration.cs

```diff
@@ -869,9 +870,18 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
 
             var pr1 = this.config.DateParser.Parse(er[0], referenceDate);
             var pr2 = this.config.DateParser.Parse(er[1], referenceDate);
-            if (pr1.Value == null || pr2.Value == null)
+
+            if (er.Count >= 2)
             {
-                return ret;
+                var dateContext = BaseDatePeriodParser.GetYearContext(this.config, er[0].Text, er[1].Text, text);

```

Comment:
This is not ideal. In regular usage BaseDatePeriodParser already has access to a config of the right type. Only this use in Chinese is strange.
It does seem it would need a big refactoring though, so I think it's OK to add this for now with a note here and in the newly static method declaration.